proto/wg/cosmo/platform/v1/platform.proto

```diff
@@ -594,6 +595,7 @@ message CreateOrganizationWebhookConfigRequest {
   string endpoint = 1;
   string key = 2;
   repeated string events = 3;
+  string events_meta = 4;
```

Comment:
Why is meta still necessary?

Reply:
I still work with meta json. It is how i form required data. The logic of storing the data has changed thats all.

Reply:
Why is it necessary? Can't we just build the data before we send the webhook?

Reply:
Thats what I do. I just send it as json.

Reply:
ok